viewing_party/party.py

```diff
@@ -1,23 +1,154 @@
 # ------------- WAVE 1 --------------------
 
-def create_movie(title, genre, rating):
-    pass
+def create_movie(movie_title, genre, rating):
+    if not movie_title or not genre or not rating:
+        return None
+    return {"title":movie_title, "genre":genre, "rating":rating}

```

Comment:
Nitpick: missing white spaces after each colon.
```suggestion
    return {"title": movie_title, "genre": genre, "rating": rating}
```